front/components/poke/PokeSlackChannelPatternInput.tsx

```diff
@@ -0,0 +1,101 @@
+import { ioTsResolver } from "@hookform/resolvers/io-ts";
+import * as t from "io-ts";
+import React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+
+import { PokeButton } from "@app/components/poke/shadcn/ui/button";
+import { InputField } from "@app/components/poke/shadcn/ui/form/fields";
+import { useSubmitFunction } from "@app/lib/client/utils";
+
+interface SlackChannelPatternInputProps {
+  initialValue: string;
+  ownerSId: string;
```

Comment:
We usually try to avoid weaken type like string for such arguments. Can we update to use `owner: LightWorkspaceType`.

Reply:
Ah yeah we definitely try to pass full types when we have them handy.

FYI: If you only have an `sId` which can happen and you want to create an interface for that we generally name it `workspaceId` for `workspace.sId`